src/cart/crt_iv.c

```diff
@@ -1953,8 +1950,6 @@ crt_ivsync_rpc_issue(struct crt_ivns_internal *ivns_internal, uint32_t class_id,
 	d_rank_list_t		excluded_list;
 	d_rank_t		excluded_ranks[1]; /* Excluding self */
 
-	IVNS_ADDREF(ivns_internal);
```

Comment:
i dont quite understand why this made any difference.
Previously we added ref every time, and decreffed either if rc != 0, or if it was not delayed completion.
For cases when completion was delayed, we decreffed when that completion was invoked.

Where was the handle leaking given this behavior?

Reply:
I was only able to narrow down the leak during daos_test -c to crt_iv_update but not the exact code path within it.

The changes in crt_ivsync_rpc_issue may not make any difference. I'm just trying to simplify the reference management.